src/analysis/fallbackDeclaration.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import {NodeAddress, EdgeAddress} from "../core/graph";
+
+import type {PluginDeclaration} from "./pluginDeclaration";
+
+import type {EdgeType, NodeType} from "./types";
+
+export const FALLBACK_NAME = "FALLBACK_ADAPTER";
+
+export const fallbackNodeType: NodeType = Object.freeze({
+  name: "node",
+  pluralName: "nodes",
+  prefix: NodeAddress.empty,
+  defaultWeight: 1,
+});
+
+export const fallbackEdgeType: EdgeType = Object.freeze({
+  forwardName: "forward edge",
```

Comment:
Aren't these supposed to be verb phrases?

What do you think of "points to"/"is pointed to by"?

Reply:
Done.